sites/page-details/includes/footer.html

```diff
@@ -0,0 +1,21 @@
+<footer id="wb-page-details-footer" class="pagedetails{% if include.addContainer %} container{% endif %}">
```

Comment:
There already is a class `.pagedetails` and so I don't think the id `#wb-page-details-footer` is needed
```suggestion
<footer class="pagedetails{% if include.addContainer %} container{% endif %}">
```
cc: @duboisp

Reply:
Changed! Thanks